README.md

```diff
@@ -10,11 +10,9 @@ the release program.*  To build your own release see [CONTRIBUTING.md](CONTRIBUT
 
 [Release](https://github.com/kubernetes-sigs/kubebuilder/releases):
 
-- [latest](https://github.com/kubernetes-sigs/kubebuilder/releases/latest)
-
 ## Book
```

Comment:
I would suggest adding `Getting Started` heading just before the book heading because "book" heading might be perceived as some long form of reading which might discourage some impatient users :)

Reply:
Done